apps/angular/src/app/features/anime-dashboard/anime-dashboard.component.ts

```diff
@@ -0,0 +1,22 @@
+import { CommonModule } from '@angular/common';
+import { Component, ChangeDetectionStrategy, inject } from '@angular/core';
+
+import { AnimeService } from '@js-camp/angular/core/services/anime.service';
+
+import { AnimeTableComponent } from './components/anime-table/anime-table.component';
+
+/** Anime list dashboard component. */
+@Component({
+	selector: 'camp-anime-dashboard',
+	templateUrl: './anime-dashboard.component.html',
+	styleUrls: ['./anime-dashboard.component.css'],
+	standalone: true,
+	imports: [CommonModule, AnimeTableComponent],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AnimeDashboardComponent {
+	private readonly animeService: AnimeService = inject(AnimeService);
+
+	/** List of anime from the API. */
```

Comment:
Too much context in docstrings. 

What if you change `getAnimeList` implementation to generate fake data? You'll have to update docstrings.

We don't care how the data comes to this component. All we need is the list itself.

Something like this will be fine:
```suggestion
	/** Anime list. */
```